code/modules/client/preferences.dm

```diff
@@ -1882,6 +1895,12 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(new_ipc_chassis)
 						features["ipc_chassis"] = new_ipc_chassis
 
+				if("ipc_brain")
+					var/new_ipc_brain
+					new_ipc_brain = input(user, "Choose your character's brain type:", "Character Preference") as null|anything in GLOB.ipc_brain_list
```

Comment:
I request, but do not required, you use tgui_input_list

Reply:
Any specific reason? No other prefs use it so it might be better to do that in one larger refactor of the system as whole.